tests/integration/server.test.js

```diff
@@ -1,28 +1,14 @@
-'use strict';
-
-var request = require('supertest');
-var server = require('../../server');
+import request from 'supertest';
+import server from '../../server';
 
 var app;
 
-describe('server tests', () => {
-    beforeEach(() => {
-        app = server.start();
-    });
+describe('INTEGRATION-TESTS::Server tests', () => {
+    beforeEach(() => app = server.start());
 
-    afterEach((done) => {
-        app.close(done);
-    });
+    afterEach((done) => app.close(done));
```

Comment:
app close does not return a promise and I didn't feel the need to promisify it.

as for () => I just thought the parens were required.